.github/workflows/deploy_latest_release.yml

```
@@ -19,10 +21,20 @@ jobs:
         uses: ./.github/actions/check_if_latest_release
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Update Github Outputs
+        id: checkRelease
+        run: |-
+          IS_LATEST_RELEASE=$(echo ${{ env.IS_LATEST_RELEASE }})
+          echo "IS_LATEST_RELEASE=${{ env.IS_LATEST_RELEASE }}" >> "$GITHUB_OUTPUT"
+          if [ $IS_LATEST_RELEASE != "true" ]; then
+            echo "Deploy is not for the latest release. Stopping deployment"
+          fi
```

How about rearranging the code in this way? If it's not the latest release, output a message. Then make sure to save the result to GITHUB_OUTPUT. 

```suggestion
          IS_LATEST_RELEASE=$(echo ${{ env.IS_LATEST_RELEASE }})
          if [ $IS_LATEST_RELEASE != "true" ]; then
            echo "Deploy is not for the latest release. Stopping deployment"
          fi
          
          echo "IS_LATEST_RELEASE=$IS_LATEST_RELEASE" >> "$GITHUB_OUTPUT"
```